client/src/components/GitHubHome.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from "react";
+import { Redirect } from "react-router-dom";
+import Styled from "styled-components";
+import { AuthContext } from "../App";
+
+
+export default function Home() {
+  const { state, dispatch } = useContext(AuthContext);
+
+  if (!state.isLoggedIn) {
+    return <Redirect to="/login" />;
+  }
+
+  const { avatar_url, name, public_repos, followers, following } = state.user
+
+  const handleLogout = () => {
+    dispatch({
+      type: "LOGOUT"
+    });
+  } 
+
+  return (
+    <Wrapper>
+      <div className="container">
+        <button onClick={()=> handleLogout()}>Logout</button>
```

Comment:
i think since we're not passing any args, it can just be `onClick={handleLogout}`